models/schema.noSQL.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+
+const reviewsSchema = new mongoose.Schema({
+  id: Number,
+  restaurant_name: String,
+  check_in: Boolean,
+  date: String,
+  stars: Number,
+  language: String,
+  review: String,
+  votes: {
+    useful: Number,
+    funny: Number,
+    cool: Number,
+  },
+  photos: [],
+  owner_fb: {
+    feedback: Boolean,
+    name: String,
+    title: String,
+    self_pic: String,
+    date: String,
+    review: String,
+  },
+  user: {
```

Comment:
Good job on just making one big collection for Mongo instead of multiple collections! You may want to refactor your object to be an array of objects though since there are multiple users to a restaurant. You can write an inner loop for each restaurant and push each object which is one user to the array of users. You can look at my photo component data generation script as an example.